.github/workflows/workflow.yaml

```diff
@@ -234,11 +234,41 @@ jobs:
           path: |
             logs-*.json
 
+  scan_vulnerabilities:
+    name: "[Trivy] Scan for vulnerabilities"
+    runs-on: ubuntu-latest
```

Comment:
Ubuntu is a pretty large image.  Considering that we run these jobs over and over, speedy jobs make for a much better development experience.

Could you try a smaller image and see if it yields a faster job?  I'm not seeing anything below that looks like it's specific to ubuntu.  Alpine or one of the thinner debian images should do the trick.

Reply:
Alpine isn't available as a [GitHub Actions virtual environment](https://github.com/actions/virtual-environments#available-environments) unless we self-host the runners 😕

Reply:
I see now that this is used all over.  No worries!